Source/KerbalCombatSystems/KerbalCombatSystems.csproj

```
@@ -9,7 +9,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>KerbalCombatSystems</RootNamespace>
     <AssemblyName>KerbalCombatSystems</AssemblyName>
-    <TargetFrameworkVersion>v4.8</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
```

```suggestion
    <TargetFrameworkVersion>v4.8</TargetFrameworkVersion>
```
I upgraded to Visual Studio 2022 after reseting my computer and it doesn't support 4.6.1. BD uses 4.8 so it's definitely safe to use. We both have to be on the same version because we can't add this file to the gitignore.

---

I reverted to 4.6.1 in the initial rush to make changes after the sudden release. I'll be ugrading versions going forward but couldn't leave it out of the pull.